docs/content/_snippets/operator/fullnode-ports.mdx

```diff
@@ -0,0 +1,12 @@
+:::info Default Firewall Ports
+
+Make sure to open these ports in your firewall configuration.  
+(These are the defaults, but may vary if you customized them at setup)
+
+| Port | Protocol | Description                                                 | Expose port ? |Reachability     |
+|------|----------|-------------------------------------------------------------|--------- |--------- |
+| 8084 | UDP      | Peer-to-peer state sync interface (external address for node to reach this node) |Needs to be reachable from the internet |Inbound / Outbound |
```

Comment:
should we add 8081 outbound to the firewall for full nodes? it's required to make transactions so the full node can connect the validators if I remember correctly